AllReadyApp/Web-App/AllReady/Views/Activity/Activity.cshtml

```diff
@@ -292,13 +301,14 @@
                         tasks = Model.Tasks,
                         skills = Model.RequiredSkills,
                         userSkills = Model.UserSkills,
-                        isVolunteeredForActivity = Model.IsUserVolunteeredForActivity
+                        isVolunteeredForActivity = Model.IsUserVolunteeredForActivity,
+                        signupModelSeed = Model.SignupModel
                     });
 
-
-        $('#phone-input').mask("(999) 999-9999");
-            
-    </script>
+        </script>
+    <script type="text/javascript" src="~/js/ko.bindings.js"></script>
+    <script type="text/javascript" src="~/lib/knockout.mapping/knockout.mapping.js"></script>

```

Comment:
You can keep using the downloaded version. You will just have to explicitly add knockout.mapping.js` to the git repo because `.gitignore` is set up to ignore everything in `~/lib`.